Sources/XcodeProj/Objects/Project/PBXProj.swift

```diff
@@ -66,6 +66,40 @@ public final class PBXProj: Decodable {
             objects: pbxproj.objects
         )
     }
+    
+    /// Initializes the project with the data representation of pbxproj file.
+    ///
+    /// - Parameters:
+    ///   - data: data representation of pbxproj file.
+    public convenience init(data: Data) throws {
+        var propertyListFormat = PropertyListSerialization.PropertyListFormat.xml
+
+        let serialized = try PropertyListSerialization.propertyList(
+            from: data,
+            options: .mutableContainersAndLeaves,
+            format: &propertyListFormat
+        )
+
+        // swiftlint:disable:next force_cast
+        let pbxProjDictionary = serialized as! [String: Any]
```

Comment:
nit: Can this just throw instead of `as!`

Reply:
@luispadron I am now throwing an error for this case and another case where this pattern is used.